lib/elixir/lib/enum.ex

```diff
@@ -1079,9 +1079,8 @@ defmodule Enum do
       {[[1], [2], [3], [4], [5]], 15}
 
   """
-  @spec flat_map_reduce(t, acc, fun) :: {[any], any}
-        when fun: (element, acc -> {t, acc} | {:halt, acc}),
-             acc: any
+  @spec flat_map_reduce(t, acc, fun) :: {[any], acc}
```

Comment:
OOC: why `[any]` instead of `list` like in `map_reduce/3` ?

Reply:
Virtually the same thing, I just left it as it was here. We could have changed it for consistency, but now that the PR has been merged the change is probably not worth it :)

Reply:
agreed